### YamlMime:ManagedReference
items:
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  id: RuntimeBinderException
  children:
  - Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor
  - Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)
  - Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: RuntimeBinderException
  nameWithType: RuntimeBinderException
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  type: Class
  summary: "Представляет ошибку, которая возникает при обработке динамической привязки в модуле привязки времени выполнения C#."
  remarks: "Исключения этого типа, отличаются от <xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException>в том, что RuntimeBinderException представляет сбой, для привязки в том смысле Ошибка компилятора обычные, тогда как <xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException>представляет неисправность связыватель времени выполнения самой.</xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException> </xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException>"
  syntax:
    content: 'public class RuntimeBinderException : Exception'
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor
  id: '#ctor'
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  langs:
  - csharp
  name: RuntimeBinderException()
  nameWithType: RuntimeBinderException.RuntimeBinderException()
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException()
  type: Constructor
  assemblies:
  - Microsoft.CSharp
  namespace: Microsoft.CSharp.RuntimeBinder
  summary: "Инициализирует новый экземпляр <xref href=&quot;Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&quot;> </xref> класса."
  remarks: "Этот конструктор инициализирует <xref:System.Exception.Message%2A>свойств нового экземпляра системным сообщением, содержащим описание ошибки, например «DefaultMessageDisplayedByParameterlessCtorWriterMustSupply» это сообщение учитывает текущую культуру системы.</xref:System.Exception.Message%2A>       В следующей таблице показаны исходные значения свойств для экземпляра класса <xref:System.InsufficientMemoryException>.</xref:System.InsufficientMemoryException>      | Свойство | Значение |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Строка локализованного сообщения об. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public RuntimeBinderException ();
    parameters: []
  overload: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  langs:
  - csharp
  name: RuntimeBinderException(String)
  nameWithType: RuntimeBinderException.RuntimeBinderException(String)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(String)
  type: Constructor
  assemblies:
  - Microsoft.CSharp
  namespace: Microsoft.CSharp.RuntimeBinder
  summary: "Инициализирует новый экземпляр <xref href=&quot;Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&quot;> </xref> класс, имеющий указанное сообщение об ошибке."
  remarks: "В следующей таблице показаны исходные значения свойств для экземпляра класса <xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException>.</xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException>      | Свойство | Значение |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Строка сообщения об ошибке, указываемой `message`. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public RuntimeBinderException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Сообщение, описывающее исключение. Код, вызывающий этот конструктор, должен убедиться, что эта строка локализована для текущего языка и региональных параметров системы."
  overload: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  langs:
  - csharp
  name: RuntimeBinderException(SerializationInfo,StreamingContext)
  nameWithType: RuntimeBinderException.RuntimeBinderException(SerializationInfo,StreamingContext)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - Microsoft.CSharp
  namespace: Microsoft.CSharp.RuntimeBinder
  summary: "Инициализирует новый экземпляр <xref href=&quot;Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&quot;> </xref> класс, который сериализованные данные."
  remarks: "Данный конструктор вызывается в процессе десериализации для воссоздания объекта исключения, переданного в потоке."
  syntax:
    content: protected RuntimeBinderException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Объект, содержащий сериализованные данные объекта о вызываемом исключении..."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Контекстные сведения об источнике или назначении."
  overload: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  langs:
  - csharp
  name: RuntimeBinderException(String,Exception)
  nameWithType: RuntimeBinderException.RuntimeBinderException(String,Exception)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(String,Exception)
  type: Constructor
  assemblies:
  - Microsoft.CSharp
  namespace: Microsoft.CSharp.RuntimeBinder
  summary: "Инициализирует новый экземпляр <xref href=&quot;Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&quot;> </xref> класс с указанным сообщением об ошибке и ссылкой на внутреннее исключение, вызвавшее данное исключение."
  remarks: "Исключение, созданное как прямой результат предыдущего исключения, должно содержать ссылку на предыдущее исключение в <xref:System.Exception.InnerException%2A>свойство.</xref:System.Exception.InnerException%2A> <xref:System.Exception.InnerException%2A>Свойство возвращает то же значение, которое передается конструктору, или `null` Если <xref:System.Exception.InnerException%2A>свойства не предоставляет конструктору значение внутреннего исключения.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       В следующей таблице показаны исходные значения свойств для экземпляра класса <xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException>.</xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException>      | Свойство | Значение |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Строка сообщения об ошибке, указываемой `message`. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public RuntimeBinderException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Сообщение об ошибке с объяснением причин исключения."
    - id: innerException
      type: System.Exception
      description: "Исключение, вызвавшее текущее исключение, или пустая ссылка, если внутреннее исключение не указано."
  overload: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException()
  nameWithType: RuntimeBinderException.RuntimeBinderException()
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException()
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException(String)
  nameWithType: RuntimeBinderException.RuntimeBinderException(String)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException(SerializationInfo,StreamingContext)
  nameWithType: RuntimeBinderException.RuntimeBinderException(SerializationInfo,StreamingContext)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException(String,Exception)
  nameWithType: RuntimeBinderException.RuntimeBinderException(String,Exception)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(String,Exception)
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException
  nameWithType: RuntimeBinderException.RuntimeBinderException
